src/default_parameters.jl

```diff
@@ -112,14 +112,15 @@ The default values of the keywords define the default model setup.
     initial_conditions::Symbol = :barotropic_vorticity    # :rest, :barotropic_vorticity or :restart
 
     # OUTPUT
-    verbose::Bool = true                # print dialog for feedback
-    output::Bool = false                # Store data in netCDF?
-    output_dt::Real = 6                 # output time step [hours]
-    output_path::String = pwd()         # path to output folder
-    output_filename::String="output.nc"     # name of the output netcdf file
-    output_vars::Vector{Symbol}=[:vor]      # variables to output: :u, :v, :vor, :div, :
-    compression_level::Integer = 3      # 1=low but fast, 9=high but slow
-    keepbits::Integer = 7               # mantissa bits to keep for every variable
+    verbose::Bool = true            # print dialog for feedback
+    output::Bool = false            # Store data in netCDF?
+    output_dt::Real = 6             # output time step [hours]
+    output_path::String = pwd()     # path to output folder
+    run_id::String = get_run_id(output, output_path)    # name of the output folder, defaults to 4-digit number counting up from 0001
```

Comment:
Do you think it's possible to make `run_id::Union{String,Integer}`? It might just be more convenient to write `run_id=1` instead of always going through `@sprintf("%04d",i)`. Meaning that if people want to use a string, fine, but if someone just wants to number runs, they can write `run_id = 1` instead of `"0001"`

Reply:
Yeah, I can add that on Monday.

Reply:
Added